### PR TITLE
Updated stress/strain header

### PR DESCRIPTION
**Pull request by fanchercm**

### CODE REVIEW

**Comment by fanchercm** [comment]
I have no idea why the test failed. i didn't push anything related to these files.

File: docs/planning/HB2B_StressStrain_peak0_Detailed.csv [hunk 1]
```diff
@@ -2,6 +2,8 @@
...
+# v: 0.25,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 vx , vy , vz , d0_1420 , d0_1420 _error , d_Dir11_1320 , d_Dir11_1320 _error , d_Dir22_1321 , d_Dir22_1321 _error , d_Dir22_1322 , d_Dir22_1322 _error , d_Dir33_1329 , d_Dir33_1329 , FWHM_Dir11_1320 , FWHM_Dir11_1320  Stress_dir33_error , FWHM_Dir22_1321 , FWHM_Dir22_1321  Strain_dir22_1322 , FWHM_Dir22_1322 , FWHM_Dir22_1322  Strain_dir22_1322  Stress_dir33_error , FWHM_Dir33_1329 , FWHM_Dir33_1329  Strain_dir33_1329 , Peak_Height_Dir11_1320 , Peak_Height_Dir11_1320  Strain_dir33_1329 , Peak_Height_Dir22_1321 , Peak_Height_Dir22_1321  Stress_dir11 , Peak_Height_Dir22_1322 , Peak_Height_Dir22_1322  Stress_dir11 _error , Peak_Height_Dir33_1329 , Peak_Height_Dir33_1329, Strain_dir11_1320 , Strain_dir11_1320  Stress_dir22 _error , Strain_dir22_1321 , Strain_dir22_1321  Stress_dir33 , Strain_dir22_1322 , Strain_dir22_1322  Stress_dir33_error , Strain_dir33_1329 , Strain_dir33_1329 , Strain_dir11 , Strain_dir11 _error , Strain_dir22 , Strain_dir22 _error , Strain_dir33 , Strain_dir33 _error , Stress_dir11 , Stress_dir11 _error , Stress_dir22 , Stress_dir22 _error , Stress_dir33 , Stress_dir33_error
```
Comment (williamfgc) [inline docs/planning/HB2B_StressStrain_peak0_Detailed.csv:7]: there is some space in the error postfix `d0_1420 _error` should be `d0_1420_error`

File: docs/planning/HB2B_StressStrain_peak0_Detailed.csv [hunk 2]
```diff
@@ -2,6 +2,8 @@
 # Direction 11 1320,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 # Direction 22: 1321 1322,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 # Direction 33: 1329,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+# E: 285,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (williamfgc) [inline docs/planning/HB2B_StressStrain_peak0_Detailed.csv:5]: right now we are spelling out the Young's modulus with units, just confirming that only E without units is the preferred choice

File: docs/planning/HB2B_StressStrain_peak0_Detailed.csv [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
+# v: 0.25,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (williamfgc) [inline docs/planning/HB2B_StressStrain_peak0_Detailed.csv:6]: right now we are spelling out Poisson's ratio, just confirming that v is the preferred choice

**Comment by williamfgc** [comment]
> I have no idea why the test failed. i didn't push anything related to these files.

@fanchercm please try `git rebase master` in your branch and force push `git push -f origin texture_user_story`.

**Pull request closed without merging**